diff -up openresty-1.15.8.1/configure.metwork openresty-1.15.8.1/configure
--- openresty-1.15.8.1/configure.metwork	2019-06-25 19:19:19.863917641 +0200
+++ openresty-1.15.8.1/configure	2019-06-25 19:24:50.982917265 +0200
@@ -727,50 +727,10 @@ _END_
             $luajit_xcflags .= " -DLUAJIT_ENABLE_GC64";
         }

-        if (!$user_luajit_xcflags
-            || $user_luajit_xcflags !~ /-msse4\.2\b/)
-        {
-            # check -msse4.2
-            my ($out, $cfile) = tempfile("resty-config-XXXXXX",
-                                         SUFFIX => '.c', TMPDIR => 1,
-                                         UNLINK => 1);
-
-            print $out "
-int main(void) {
-#ifndef __SSE4_2__
-#   error SSE 4.2 not found
-#endif
-    return 0;
-}
-";
-            close $out;
-
-            my $ofile = tmpnam();
-            my $comp = ($cc || 'cc');
-            my $found;
-
-            if (system("$comp -o $ofile -msse4.2 -c $cfile") == 0
-                && -s $ofile)
-            {
-                unlink $ofile;
-
-                if (system("$comp -o $ofile -march=native -c $cfile") == 0
-                    && -s $ofile)
-                {
-                    print "INFO: found -msse4.2 in $comp.\n";
-                    $found = 1;
-                    $luajit_xcflags .= " -msse4.2";
-                }
-            }
-
-            if (-f $ofile) {
-                unlink $ofile;
-            }
-
-            if (!$found) {
-                print "WARNING: -msse4.2 not supported in $comp.\n";
-            }
-        }
+        # We don't want to use sse42 optimisations
+        # for the moment
+        # see https://github.com/openresty/openresty/issues/267
+        # https://github.com/metwork-framework/mfext/issues/384

         if ($opts->{debug}) {
             $luajit_xcflags .= " -DLUA_USE_APICHECK -DLUA_USE_ASSERT";
