diff -up openresty-1.15.8.3/bundle/nginx-1.15.8/src/core/nginx.h.orig openresty-1.15.8.3/bundle/nginx-1.15.8/src/core/nginx.h
--- openresty-1.15.8.3/bundle/nginx-1.15.8/src/core/nginx.h.orig        2025-07-09 16:27:14.762281243 +0000
+++ openresty-1.15.8.3/bundle/nginx-1.15.8/src/core/nginx.h     2025-07-09 16:27:28.882294677 +0000
@@ -11,7 +11,7 @@

 #define nginx_version      1015008
 #define NGINX_VERSION      "1.15.8"
-#define NGINX_VER          "openresty/" NGINX_VERSION ".3"
+#define NGINX_VER          "openresty/" NGINX_VERSION ".4"

 #ifdef NGX_BUILD
 #define NGINX_VER_BUILD    NGINX_VER " (" NGX_BUILD ")"

