diff -up certifi-2019.3.9/certifi/core.py.orig certifi-2019.3.9/certifi/core.py
--- certifi-2019.3.9/certifi/core.py.orig	2019-06-11 16:55:03.975264582 +0200
+++ certifi-2019.3.9/certifi/core.py	2019-06-11 17:05:13.816264186 +0200
@@ -7,9 +7,22 @@ certifi.py
 This module returns the installation location of cacert.pem.
 """
 import os
+import platform
+
+
+def is_centos_rhel():
+    try:
+        distribution_name = \
+            platform.linux_distribution()[0].lower().replace(' ', '')
+    except Exception:
+        return False
+    return ( ('centos' in distribution_name) or ('redhat' in distribution_name))
 
 
 def where():
+    if is_centos_rhel():
+        return "/etc/pki/ca-trust/extracted/pem/tls-ca-bundle.pem"
+
     f = os.path.dirname(__file__)
 
     return os.path.join(f, 'cacert.pem')
