diff -up certifi-2022.6.15.1/certifi/core.py.orig certifi-2022.6.15.1/certifi/core.py
--- certifi-2022.6.15.1/certifi/core.py.orig	2022-09-13 11:22:26.674138774 +0000
+++ certifi-2022.6.15.1/certifi/core.py	2022-09-13 11:28:21.061398598 +0000
@@ -5,6 +5,17 @@ certifi.py
 This module returns the installation location of cacert.pem or its contents.
 """
 import sys
+import distro
+
+def is_centos_rhel_rocky():
+    try:
+        distribution_name = distro.id()
+    except Exception:
+        return False
+    return ( ('centos' in distribution_name) or
+             ('redhat' in distribution_name) or
+             ('rocky' in distribution_name) )
+
 
 
 if sys.version_info >= (3, 9):
@@ -15,6 +26,9 @@ if sys.version_info >= (3, 9):
     _CACERT_PATH = None
 
     def where() -> str:
+        if is_centos_rhel_rocky():
+            return "/etc/pki/ca-trust/extracted/pem/tls-ca-bundle.pem"
+
         # This is slightly terrible, but we want to delay extracting the file
         # in cases where we're inside of a zipimport situation until someone
         # actually calls where(), but we don't want to re-extract the file
@@ -49,6 +63,9 @@ elif sys.version_info >= (3, 7):
     _CACERT_PATH = None
 
     def where() -> str:
+        if is_centos_rhel_rocky():
+            return "/etc/pki/ca-trust/extracted/pem/tls-ca-bundle.pem"
+
         # This is slightly terrible, but we want to delay extracting the
         # file in cases where we're inside of a zipimport situation until
         # someone actually calls where(), but we don't want to re-extract
@@ -100,6 +117,9 @@ else:
     # If we don't have importlib.resources, then we will just do the old logic
     # of assuming we're on the filesystem and munge the path directly.
     def where() -> str:
+        if is_centos_rhel_rocky():
+            return "/etc/pki/ca-trust/extracted/pem/tls-ca-bundle.pem"
+
         f = os.path.dirname(__file__)
 
         return os.path.join(f, "cacert.pem")
