diff -up certifi-2019.3.9/certifi/core.py.orig certifi-2019.3.9/certifi/core.py
--- certifi-2019.3.9/certifi/core.py.orig	2022-01-17 16:22:19.396418680 +0100
+++ certifi-2019.3.9/certifi/core.py	2022-01-17 16:28:20.836130276 +0100
@@ -7,9 +7,22 @@ certifi.py
 This module returns the installation location of cacert.pem.
 """
 import os
+import distro
+
+def is_centos_rhel_rocky():
+    try:
+        distribution_name = distro.id()
+    except Exception:
+        return False
+    return ( ('centos' in distribution_name) or
+             ('redhat' in distribution_name) or
+             ('rocky' in distribution_name) )
 
 
 def where():
+    if is_centos_rhel_rocky():
+        return "/etc/pki/ca-trust/extracted/pem/tls-ca-bundle.pem"
+
     f = os.path.dirname(__file__)
 
     return os.path.join(f, 'cacert.pem')
