diff -up jasper-2.0.14/CMakeLists.txt.orig jasper-2.0.14/CMakeLists.txt
--- jasper-2.0.14/CMakeLists.txt.orig	2018-05-23 14:49:17.799956450 +0200
+++ jasper-2.0.14/CMakeLists.txt	2018-05-23 14:49:36.026956440 +0200
@@ -1,6 +1,6 @@
 cmake_minimum_required (VERSION 2.8.11)
 
-project(JasPer LANGUAGES C)
+project(JasPer C)
 
 set(CMAKE_MODULE_PATH
   ${CMAKE_MODULE_PATH} "${CMAKE_SOURCE_DIR}/build/cmake/modules/")
@@ -293,7 +293,7 @@ if (JAS_ENABLE_OPENGL AND OPENGL_FOUND)
 		endif()
 	endif()
 	# On some systems (e.g., Fedora 21), there is a bug in the cmake code
-	# that detects GLUT libraries.  The following ugliness is a workaround for 
+	# that detects GLUT libraries.  The following ugliness is a workaround for
 	# this problem.
 	if (NOT GLUT_Xmu_LIBRARY OR NOT GLUT_Xmi_LIBRARY)
 		if (NOT GLUT_Xmu_LIBRARY)
@@ -345,7 +345,7 @@ if (JAS_ENABLE_SHARED)
 
 	# when building, don't use the install RPATH already
 	# (but later on when installing)
-	set(CMAKE_BUILD_WITH_INSTALL_RPATH FALSE) 
+	set(CMAKE_BUILD_WITH_INSTALL_RPATH FALSE)
 
 	set(CMAKE_INSTALL_RPATH "${CMAKE_INSTALL_PREFIX}/lib")
 
