diff -up certifi-2018.4.16/certifi/core.py.orig certifi-2018.4.16/certifi/core.py
--- certifi-2018.4.16/certifi/core.py.orig	2018-07-09 14:41:03.129650136 +0200
+++ certifi-2018.4.16/certifi/core.py	2018-07-09 14:50:44.747805364 +0200
@@ -8,9 +8,19 @@ certifi.py
 This module returns the installation location of cacert.pem.
 """
 import os
+import platform
 import warnings
 
 
+def is_centos_rhel():
+    try:
+        distribution_name = \
+            platform.linux_distribution()[0].lower().replace(' ', '')
+    except Exception:
+        return False
+    return ( ('centos' in distribution_name) or ('redhat' in distribution_name))
+
+
 class DeprecatedBundleWarning(DeprecationWarning):
     """
     The weak security bundle is being deprecated. Please bother your service
@@ -19,6 +29,9 @@ class DeprecatedBundleWarning(Deprecatio
 
 
 def where():
+    if is_centos_rhel():
+        return "/etc/pki/ca-trust/extracted/pem/tls-ca-bundle.pem"
+
     f = os.path.dirname(__file__)
 
     return os.path.join(f, 'cacert.pem')
