diff -up redis-5.0.5/src/config.h.orig redis-5.0.5/src/config.h
--- redis-5.0.5/src/config.h.orig	2019-07-12 11:38:15.271920521 +0200
+++ redis-5.0.5/src/config.h	2019-07-12 11:38:40.863920493 +0200
@@ -116,20 +116,6 @@
 #define rdb_fsync_range(fd,off,size) fsync(fd)
 #endif
 
-/* Check if we can use setproctitle().
- * BSD systems have support for it, we provide an implementation for
- * Linux and osx. */
-#if (defined __NetBSD__ || defined __FreeBSD__ || defined __OpenBSD__)
-#define USE_SETPROCTITLE
-#endif
-
-#if ((defined __linux && defined(__GLIBC__)) || defined __APPLE__)
-#define USE_SETPROCTITLE
-#define INIT_SETPROCTITLE_REPLACEMENT
-void spt_init(int argc, char *argv[]);
-void setproctitle(const char *fmt, ...);
-#endif
-
 /* Byte ordering detection */
 #include <sys/types.h> /* This will likely define BYTE_ORDER */
 
