diff -up proj-6.2.1/src/proj_api.h.orig proj-6.2.1/src/proj_api.h
--- proj-6.2.1/src/proj_api.h.orig	2019-12-03 16:41:28.327936507 +0100
+++ proj-6.2.1/src/proj_api.h	2019-12-03 16:40:11.326936601 +0100
@@ -33,8 +33,9 @@
   *
   */
 
+/* We allow use of this deprecated file in Metwork because of libspatialite and cartopy */
 #ifndef ACCEPT_USE_OF_DEPRECATED_PROJ_API_H
-#error 'To use the proj_api.h you must define the macro ACCEPT_USE_OF_DEPRECATED_PROJ_API_H'
+#warning 'proj_api.h is deprecated and will be removed in version 7, you should use proj.h'
 #endif
 
 #ifndef PJ_VERSION