diff -up wrk2-master/Makefile.orig wrk2-master/Makefile
--- wrk2-master/Makefile.orig	2019-11-08 14:24:20.297486225 +0100
+++ wrk2-master/Makefile	2019-11-08 14:26:27.465486082 +0100
@@ -14,9 +14,13 @@ else ifeq ($(TARGET), darwin)
 	LIBS += -L/usr/local/opt/openssl/lib
 	CFLAGS += -I/usr/local/include -I/usr/local/opt/openssl/include
 else ifeq ($(TARGET), linux)
-        CFLAGS  += -D_POSIX_C_SOURCE=200809L -D_BSD_SOURCE
+	CFLAGS  += -D_POSIX_C_SOURCE=200809L -D_BSD_SOURCE
 	LIBS    += -ldl
 	LDFLAGS += -Wl,-E
+	ifneq ($(OPENSSL_HOME), )
+		CFLAGS  += -I$(OPENSSL_HOME)/include
+		LDFLAGS += -L$(OPENSSL_HOME)/lib
+	endif
 else ifeq ($(TARGET), freebsd)
 	CFLAGS  += -D_DECLARE_C99_LDBL_MATH
 	LDFLAGS += -Wl,-E
